File: santhosh/preraktiwari_fl-ss_backend-05de6a7bde97/preraktiwari_fl-ss_backend-05de6a7bde97/preraktiwari_fl-ss_backend-05de6a7bde97/src/main/java/com/surshree/app/models/training/TrainingModel.java
package com.surshree.app.models.training;

import lombok.Data;
import lombok.NoArgsConstructor;

import java.math.BigDecimal;
import java.util.List;
import java.util.UUID;

@Data
@NoArgsConstructor
public class TrainingModel {
    private Long trainingId;

    private List<TrainingContentModel> trainingContents;

    private String category;

    private String trainingName;

    private String trainingDesc;

    private String isSubscriptionRequired;

    private BigDecimal subscriptionAmount;

    private String trainingPhoto;

    private Boolean isSubscribed = Boolean.FALSE;
}

File: santhosh/fixed frontend/src/app/models/TrainingModel.ts
import { TrainingContentModel } from './TrainingContentModel';

export class TrainingModel {
    trainingId: string;

    trainingContents: TrainingContentModel[];

    category: string;

    trainingName: string;

    trainingDesc: string;

    isSubscriptionRequired: string;

    subscriptionAmount: string;

    trainingPhoto: string;

    isSubscribed : boolean;
}